composer.json

```
@@ -66,7 +66,8 @@
     },
     "conflict": {
         "coduo/php-matcher": "6.0.12",
-        "doctrine/persistence": "1.3.2"
+        "doctrine/persistence": "1.3.2",
+        "doctrine/orm": "2.10.0 || 2.14.2 || >=2.16.0"
```

Should not longer be required

```suggestion
        "doctrine/persistence": "1.3.2"
```